WordPress/Classes/ViewRelated/Post/FeaturedImageViewController.m

```diff
@@ -94,7 +100,11 @@ - (void)hideBars:(BOOL)hide animated:(BOOL)animated
         if (hide) {
             self.view.backgroundColor = [UIColor blackColor];
         } else {
-            self.view.backgroundColor = [UIColor whiteColor];
+            if (@available(iOS 13, *)) {
+                self.view.backgroundColor = [UIColor systemBackgroundColor];
```

Comment:
As above, we can use `basicBackground` here.

Reply:
Ah, right, thank you! I will update it to use basicBackground, and eliminate those other accidental changes.

Reply:
Great, thank you – let me know when it's ready for another look!